ansible_collections/arista/avd/plugins/plugin_utils/ip_addressing_utils.py

```diff
@@ -0,0 +1,17 @@
+import ipaddress
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+
+
+def _ip(pool: str, prefixlen: int, subnet_offset: int, ip_offset: int) -> str:
```

Comment:
maybe call it `get_ip_from_pool` or something?

Reply:
sure, will change it.